Documentation/README_ObjectManipulator.md

```diff
@@ -8,9 +8,14 @@ The *ObjectManipulator* script makes an object movable, scalable, and rotatable
 
 ## How to use the object manipulator
 
-To use the object manipulator, first add the [`ObjectManipulator.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Experimental/Features/Input/Handlers/ObjectManipulator.cs) component to a GameObject.
+To use the object manipulator, first add the `ObjectManipulator` script component to a GameObject:
 
-Make sure to also add a collider to the object, matching its grabbable bounds. To make the object respond to near articulated hand input, add the [`NearInteractionGrabbable.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionGrabbable.cs) script as well.
+- Selecting the GameObject in the Hierarchy
```

Comment:
do we really need those additional steps that describe how to add a component to a gameobject ? i think users should know that or look that up in the unity docs if they don't know - else we would have to put this in all our doc pages to stay consistent

Reply:
This was a targeted update added to address the rename of the services folder. I was planning on putting similar instructions to other docs as I rename other folders.

I am happy to remove these or leave as is.

Reply:
removed the additional steps.